app/lib/document_navigation.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class DocumentNavigation
```

Comment:
Looks like we pulled controllerNavigation into a concern when we started having multiple "flows", but we never imported the concern into here and had most of the methods duplicated. Where possible, I removed the duplicated methods.